TestCases/turbomachinery/axial_stage_full_annulus_unsteady/rotor_FA_Unsteady.cfg

```diff
@@ -0,0 +1,22 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: 2D Axial stage                                             %
+% Author: S. Vitale                                                            %
+% Institution: Delft University of Technology                                  %
+% Date: Feb 28th, 2017                                                         %
+% File Version 5.0.0"cardinal"                                                 %
```

Comment:
Change authorship and get the credit that you deserve... Can you also add all new testcases to the regression tests, at least put it into parallel_regression.py